api/src/functions/graphql/package-lock.json

```diff
@@ -1,2618 +1,8 @@
 {
   "name": "grand-stack-starter-api-graphql-serverless",
   "version": "0.0.1",
-  "lockfileVersion": 2,
```

Comment:
I think my question about this would be that, why are there so many changs in this file?

Reply:
not even sure, i was also wondering why, i will use the file from the deploy branch to make sure there are no changes